src/components/PlayerView/index.tsx

```diff
@@ -35,9 +35,23 @@ export interface PlayerViewProps extends BasePlayerViewProps, PlayerViewEvents {
    */
   player: Player;
 
+  /**
+   * The `FullscreenHandler` that is used by the `PlayerView` to control the fullscreen mode.
+   */
   fullscreenHandler?: FullscreenHandler;
 
+  /**
+   * The `CustomMessageHandler` that can be used to directly communicate with the embedded WebUi.
+   */
   customMessageHandler?: CustomMessageHandler;
+
+  /**
+   * Can be set to `true` to enter fullscreen mode, or `false` to exit fullscreen mode.
+   * By using this property to change the fullscreen state, we make sure that the embedded Player UI is also aware
+   * of potential fullscreen state changes.
+   * To use this property, a `FullscreenHandler` must be set.
```

Comment:
Should we mention that for state changes they should listen to `onFullscreenEnter` and `onFullscreenExit` events?

Reply:
I would say no. The fullscreen state is a state of the screen the `PlayerView` is part of. So the screen should know the state independent of the `PlayerView`.

Reply:
@matamegger What I had in mind is that this value will get out of sync once you hit the fullscreen button on the player UI.

Reply:
https://github.com/bitmovin/bitmovin-player-react-native/pull/169/commits/05b9afa1b853cc838ae2b4801dc9cb0f9929b4ce

Rethinking this, having an explicit note is probably best.